lib/constants/dashboard.js

```diff
@@ -1,3 +1,7 @@
 export const JWT_SECURED = 'JWT_SECURED';
 export const ANY = 'ANY';
 export const OFF = 'OFF';
+
+export const BLANK_DASHBOARD = 'blankDashboard';
+export const CURATR_STARTER = 'curatrStarter';
```

Comment:
@vladislav-999 The change from Curatr to Stream has been merged (https://github.com/LearningLocker/enterprise/releases/tag/v6.3.4) so can all the references to Curatr in this pr be updated to Stream please.

```suggestion
export const STREAM_STARTER = 'streamStarter';
```